weave-js/src/components/PagePanelComponents/Home/Browse3/pages/wfReactInterface/tsDataModelHooksEvaluationComparison.ts

```diff
@@ -548,6 +482,532 @@ const fetchEvaluationComparisonResults = async (
     evalCall => evalCall.modelRef
   );
 
+  // If we have processed imperative evaluations, we can skip the following code
+  // as it's meant for non-imperative evaluations
+  if (!isImperativeEval) {
+    populatePredictionsAndScoresNonImperative(
+      evalTraceRes,
+      result,
+      summaryData,
+      summaryOps,
+      scorerRefs,
+      predictAndScoreOps,
+      modelRefs,
+      convertISOToDate,
+      evaluationCallIds
+    );
+  }
+
+  // Filter out non-intersecting rows
+  result.resultRows = Object.fromEntries(
+    Object.entries(result.resultRows).filter(([digest, row]) => {
+      return (
+        Object.values(row.evaluations).length ===
+        Object.values(summaryData.evaluationCalls).length
+      );
+    })
+  );
+
+  return result;
+};
+
+/// Non exported helpers below
+
+const modelLatencyMetricDimension: MetricDefinition = {
```

Comment:
given the magnitude of the change, it would be nice if symbols like this didn't move for easier identification of changes

Reply:
yeah it's unfortunate git made the cut like this